### PR TITLE
Actual auto-gen'ed code

### PR DESCRIPTION
**Pull request by chrisdunelm**

### CODE REVIEW

File: src/Google.Pubsub.V1/PublisherClient.cs [hunk 1]
```diff
@@ -97,15 +97,22 @@ public static class ServiceDefaults
         }
 
         /// <summary>
-        /// Path template for a project resource. Parameters:
+        /// Path template for a Project resource. Parameters:
```
Comment (jskeet) [inline src/Google.Pubsub.V1/PublisherClient.cs]: We should consider lower-casing the resource name to make it read better. Possibly add a TODO in the codegen?

File: src/Google.Pubsub.V1/PublisherClient.cs [hunk 2]
```diff
@@ -199,6 +204,20 @@ public virtual Publisher.IPublisherClient GrpcClient
             get { throw new NotImplementedException(); }
         }
 
+        /// <summary>
```
Comment (jskeet) [inline src/Google.Pubsub.V1/PublisherClient.cs]: It's great to see all this documentation. Will run docfx on it and update my github pages when it's merged.

File: src/Google.Pubsub.V1/SubscriberClient.cs [hunk 3]
```diff
@@ -147,7 +146,7 @@ public static class ServiceDefaults
         };
 
         /// <summary>
-        /// Asynchronously create a <see cref="SubscriberClient"/> from default credentials.
+        /// Asynchonously create a <see cref="SubscriberClient"/> from default credentials.
```
Comment (jskeet) [inline src/Google.Pubsub.V1/SubscriberClient.cs]: Asynchronously

**Comment by jskeet** [comment]
LGTM beyond nits. Looking forward to seeing what happens when we run this on other services.

File: src/Google.Pubsub.V1/PublisherClient.cs (hunk 1 above)
Comment (chrisdunelm) [inline src/Google.Pubsub.V1/PublisherClient.cs]: Resource name now lower-cased in summary

File: src/Google.Pubsub.V1/SubscriberClient.cs (hunk 3 above)
Comment (chrisdunelm) [inline src/Google.Pubsub.V1/SubscriberClient.cs]: Done

File: src/Google.Pubsub.V1/PublisherClient.cs (hunk 2 above)
Comment (chrisdunelm) [inline src/Google.Pubsub.V1/PublisherClient.cs]: :)

**Pull request merged**